### PR TITLE
Combiner LsPci should not modify Parser LsPci*

### PR DESCRIPTION
**Pull request by xiangce**
Signed-off-by: Xiangce Liu <xiangceliu@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [x] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Fixes: #3147

### CODE REVIEW

**Comment by xiangce** [comment]
@bfahr - would you please to have a look at this MR whenever free.

It's trying to fix a bug (1966874) reported by QE

Thanks.

File: insights/combiners/lspci.py [hunk 1]
```diff
@@ -115,15 +115,17 @@ class LsPci(list):
     def __init__(self, lspci_k, lspci_vmmkn):
         if lspci_vmmkn:
             for dev in lspci_vmmkn:
+                dev = dev.copy()
```
Comment (ryan-blakley) [inline insights/combiners/lspci.py:118]: @xiangce would it be possible to add a comment for the copies to explain why they're needed, to help prevent it from possibly happening again in the future?

Comment (xiangce) [reply insights/combiners/lspci.py:119]: @ryan-blakley yes, comment should be added. Please have a check again. Thanks.

Comment (ryan-blakley) [reply insights/combiners/lspci.py:119]: @xiangce Thanks, it looks good to me.

**Review by bfahr — APPROVED** [review]

**Pull request merged**